packages/client-config/API.md

```diff
@@ -627,7 +627,7 @@ export type CustomClientConfig = {
 export const DEFAULT_CLIENT_CONFIG_VERSION: ClientConfigVersion;
 
 // @public
-export const generateClientConfig: <T extends "1" | "1.1" | "1.2" | "1.3" | "0">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{
+export const generateClientConfig: <T extends "1.3" | "0" | "1" | "1.1" | "1.2">(backendIdentifier: DeployedBackendIdentifier, version: T, awsClientProvider?: AWSClientProvider<{

```

Comment:
the changes in `packages/client-config/API.md` and `packages/platform-core/API.md` are unexpected.

You can either revert these two files.
Or run `npm run clean && npm install && npm run build && npm update:api` . I.e. re-generate api reports from clean workspace.